comps/guardrails/pii_detection/pii/detect/emails_detection.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+""" This code is adapted from BigScience PII detection
```

Comment:
@chensuyue  could you pls check if it's right license format for codes derived from 3rd party?

Reply:
We had this file approved by OSPDT in our own project, by claiming the source from another opensource project is suggested by OSPDT committee. I'll resolve this issue.